db-tests/src/test/java/gov/nasa/jpl/aerie/database/DatabaseTestHelper.java

```diff
@@ -84,31 +105,154 @@ public void startDatabase() throws SQLException, IOException, InterruptedExcepti
     connection = hikariDataSource.getConnection();
   }
 
+  /**
+   * Runs the given sql file
+   */
+  public void runSqlFile(final String filePath) throws IOException, InterruptedException, SQLException {
+    final var errors = runSubprocess(
+        "psql",
+       "postgresql://postgres:postgres@localhost:5432/" + dbName,
+       "-v", "ON_ERROR_STOP=1",
+       "-f", filePath);
+    if (errors.toLowerCase().contains("error:")) {
```

Comment:
TODO: Maybe we should check the return code of psql instead of (or in addition to) looking for the string "errors:" in the output.